assets/scss/hero.scss

```diff
@@ -255,3 +232,33 @@
     }
   }
 }
+
+@media (max-width: $bp-medium) {
+  .hero-wrapper {
+    padding: 0;
+    margin-top: -1rem;
+    .jumbotron.browse-jumbotron {
+      margin-top: 0;
+      background-size: 56.5rem;
+      background-position: center top;
+    }
+  }
+}
+
+@media (max-width: $bp-small) {
+  .hero-wrapper {
+    .jumbotron.browse-jumbotron {
+      border-radius: 0;
+      .card {
+        margin: 0;
+        border-radius: 0;
+        h1 {
+          font-size: 1rem;
+        }
+        p, .btn {
+          font-size: 0.875rem;
+        }
```

Comment:
There is a comment in Zeplin that says "On mobile the card should expand vertically from the top.
•	The image is always the same size / ratio." 

When I now take a look at the current implementation on mobile, an image would hardly be visible if there is a lot of text.

Reply:
well the mobile image needed to be set at a fixed height since the image is the same one used on desktop, if it was larger/used % based width then we would see tiling effects as the image grew. If the h1/p text can be limited then we can avoid having the image hidden